src/popup/components/TransactionDetailsBase.vue

```diff
@@ -0,0 +1,473 @@
+          :label="$t('pages.transactionDetails.gasUsed')"
+          data-cy="gas"
+        />
+        <DetailsItem
+          v-if="transactionFee"
+          :label="$t('transaction.fee')"
+          data-cy="fee"
+        >
+          <template #value>
+            <TokenAmount
+              :amount="transactionFee"
+              :symbol="coinSymbol"
+              :protocol="protocol"
+              :high-precision="!!noneAeCoin"
+            />
+          </template>
+        </DetailsItem>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+
+import { PROTOCOL_AETERNITY } from '@/constants';
+import type { Protocol, ITransaction } from '@/types';
+import {
+  formatDate,
+  formatTime,
+  splitAddress,
+} from '@/utils';
+
+import TransactionOverview from '@/popup/components/TransactionOverview.vue';
+import SwapRoute from '@/popup/components/SwapRoute.vue';
+import SwapRates from '@/popup/components/SwapRates.vue';
+import TokenAmount from '@/popup/components/TokenAmount.vue';
+import DetailsItem from '@/popup/components/DetailsItem.vue';
+import LinkButton from '@/popup/components/LinkButton.vue';
+import Truncate from '@/popup/components/Truncate.vue';
+import TransactionTokens from '@/popup/components/TransactionTokenRows.vue';
+import CopyText from '@/popup/components/CopyText.vue';
+import TransactionDetailsPoolTokens from '@/popup/components/TransactionDetailsPoolTokens.vue';
+import PayloadDetails from '@/popup/components/PayloadDetails.vue';
+import TransactionErrorStatus from '@/popup/components/TransactionErrorStatus.vue';
+import Avatar from '@/popup/components/Avatar.vue';
+import DialogBox from '@/popup/components/DialogBox.vue';
+
+import AnimatedPending from '@/icons/animated-pending.svg?vue-component';
+import ExternalLink from '@/icons/external-link.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    PayloadDetails,
+    TransactionErrorStatus,
+    TransactionDetailsPoolTokens,
+    TransactionTokens,
+    TransactionOverview,
+    TokenAmount,
+    DetailsItem,
+    LinkButton,
+    Truncate,
+    CopyText,
+    SwapRoute,
+    SwapRates,
+    AnimatedPending,
+    ExternalLink,
+    Avatar,
+    DialogBox,
+  },
+  props: {
+    transaction: { type: Object as PropType<ITransaction | undefined>, required: true },
+    coinSymbol: { type: String, required: true },
+    transactionFee: { type: Number, required: true },
+    isSwap: { type: Boolean },
+    isPool: { type: Boolean },
+    tokenSymbol: { type: String, required: true },
+    totalAmount: { type: Number, required: true },
+    isErrorTransaction: { type: Boolean },
+    isDexAllowance: { type: Boolean },
+    isDex: { type: Boolean },
+    isTransactionAex9: { type: Boolean },
+    payload: { type: String, default: '' },
+    isMultisig: { type: Boolean },
+    tipUrl: { type: String, default: '' },
+    tipLink: { type: String, default: '' },
```

Comment:
Tipping is also related to AE directly, so maybe there is a better way of displaying tipping data without having these props?